tools/hansel/bio_hansel.xml

```diff
@@ -1,33 +1,29 @@
-<tool id="bio_hansel" name="Hansel - Salmonella Subtyping" version="0.1.0">
+<tool id="bio_hansel" name="Salmonella Subtyping" version="0.1.1">
+    <description>Subtype *Salmonella* genome assemblies (FASTA files) and/or whole-genome sequencing reads (FASTQ files)!</description>
```

Comment:
Get rid of "Subtype *Salmonella*" since the name value will be there already.  Also remove both (FASTA files) and (FASTQ files) since the end user will already know this.